test/test_argparse.py

```diff
@@ -12,6 +12,10 @@
 class TestParseItem(HPyTest):
     def make_parse_item(self, fmt, type, hpy_converter):
         mod = self.make_module("""
+            #define CHAR_TO_HPYBYTES(ctx, a) ( \
```

Comment:
just nitpick, but what about `static inline` instead of `#define`?

Reply:
Done!